src/main/java/cz/cvut/kbss/termit/rest/UserController.java

```diff
@@ -106,4 +109,15 @@ public void disable(@PathVariable(name = "fragment") String identifierFragment)
     public Boolean exists(@RequestParam(name = "username") String username) {
         return userService.exists(username);
     }
-}
+
+    @PreAuthorize("hasRole('" + SecurityConstants.ROLE_ADMIN + "')")
+    @PostMapping(value = "/{fragment}/role",
```

Comment:
Wouldn't PUT be more suitable here? Since we are not adding a role, we are updating it for the user...